packages/cloudflare/src/cli/build/patches/plugins/use-cache.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * This patch will replace the createSnapshot function in the
+ * server/app-render/async-local-storage.js file to an empty string.
+ * This is necessary because the createSnapshot function is causing I/O issues for
+ * ISR/SSG revalidation in Cloudflare Workers.
+ * TODO: Find a better fix for this issue.
+ */
```

Comment:
Could you please add more details from the PR discussion:

https://github.com/vercel/next.js/blob/62de494f/packages/next/src/server/app-render/async-local-storage.ts#L55-L65

> This is because by default it will use AsyncLocalStorage.snapshot() and it will bind everything to the initial request context. The downsides is that use cache function will have access to the full request ALS context from next (i.e. cookies, headers ...)